code/dijkstra-algorithm/src/main/java/ru/unn/agile/DijkstraAlgorithm/Model/DijkstraGraph.java

```diff
@@ -0,0 +1,149 @@
+package ru.unn.agile.DijkstraAlgorithm.Model;
+
+import java.util.*;
```

Comment:
Use concrete imports instead of using *.

Reply:
fixed